tools/downloader/topologies.py

```diff
@@ -0,0 +1,66 @@
+import os
+import argparse
+import sys
+
+from pathlib import Path
+from argparse import Namespace
+
+import common
+import downloader
+import converter
+import info_dumper
+
+_cache = Path(os.getenv('OPEN_MODEL_ZOO_CACHE_DIR', Path('.').resolve() / 'models'))
+
+def load_model(model_name, precision='FP32'):
+    parser = argparse.ArgumentParser()
+    args = Namespace(all=False, list=None, name=model_name, print_all=False)   
+    _topology = common.load_models_from_args(parser, args)[0]
```

Comment:
Inside the method you can avoid using underscore (it's used to hide some methods or variables from other scripts where `topologies.py` is imported)